src/jobsautoreport/slack.py

```diff
@@ -198,9 +198,11 @@ def _upload_most_failing_jobs_graph(
         file_title: str,
         thread_time_stamp: Optional[str],
     ) -> None:
-        names = [job[0] for job in jobs]
-        successes = [job_states_count.successes for _, job_states_count in jobs]
-        failures = [job_states_count.failures for _, job_states_count in jobs]
+        names = [job[0] for job in reversed(jobs)]
```

Comment:
As discussed, let's do the sorting in the reporter and remove those ``reversed`` calls from here 
This way the unit-tests can prevent regressions of this logic